### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by AlanE23**
Submitting my Javascript quiz for review, thanks.

### CODE REVIEW

File: quiz.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+var willisTField = document.getElementById("bHeight");
+
+princeField.addEventListener("keypress", enterKey);
+
+willisTField.addEventListener("keypress", enterKey);
+
+function enterKey (e) {
+  var pressKey = e.keyCode;
+  if (pressKey==13) {
```
Comment (gregkorte) [inline quiz.js]: Remember to use `===` when comparing things in js.

File: quiz.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  for (var i = 1; i <= peyton.willisT; i++) {
+
+    // assigning a value to planet, height of tree - i
+    planet = peyton.willisT - i;
+
+    // assigning a value to numP, proportional to i, always odd for symmetry
+    numP = (2 * i) - 1;
+
+    tree += space.repeat(planet) + peyton.prince.repeat(numP) + "\n";
```
Comment (gregkorte) [inline quiz.js:69]: By appending concats `+=` and not resetting `tree` this line causes strange behavior when running the program concurrently.

**Comment by gregkorte** [comment]
Nice job on your quiz, Alan. See the above comments. For bonus try to do some error checking for letters in the height input and more than one character in the symbol input.

**Comment by AlanE23** [comment]
Fixed issues and added the error checking for the input fields

**Comment by gregkorte** [comment]
Nice work! Going forward, try to give your variables names that will make sense to anyone who reads your code. Good to merge. :+1:

**Pull request merged**